src/actions/index.js

```diff
@@ -102,6 +105,14 @@ export const types = {
   SHOW_GLOBAL_MODAL: 'SHOW_GLOBAL_MODAL',
   HIDE_GLOBAL_MODAL: 'HIDE_GLOBAL_MODAL',
   SERVER_INFO_UPDATED: 'SERVER_INFO_UPDATED',
+  REOWN_SEND_TX_STATUS_LOADING: 'REOWN_SEND_TX_STATUS_LOADING',
+  REOWN_SEND_TX_STATUS_READY: 'REOWN_SEND_TX_STATUS_READY',
+  REOWN_SEND_TX_STATUS_SUCCESS: 'REOWN_SEND_TX_STATUS_SUCCESS',
+  REOWN_SEND_TX_STATUS_FAILURE: 'REOWN_SEND_TX_STATUS_FAILURE',
```

Comment:
In all other RPC requests you use `FAILED` instead of `FAILURE`, e.g., `REOWN_NEW_NANOCONTRACT_STATUS_FAILED`. I think we should keep consistency

Reply:
I agree, refactored, thanks